doc/lua_api.txt

```diff
@@ -234,6 +235,40 @@ stripping out the file extension:
 * e.g. `foomod_foothing.png`
 * e.g. `foomod_foothing`
 
+Texture pack directory structure
```

Comment:
Seems the logical place for it, seeing as 'mod directory structure' is documented close above at https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L126